compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/Constants.java

```diff
@@ -47,4 +47,7 @@ private Constants() {
     public static final int MAX_UNICODE = 0x10FFFF;
 
     public static final String DESUGARED_MAPPING_CONSTR_KEY = "$mapping$var$";
+
+    public static final String REMOVE_IF_HAS_KEY = "removeIfHasKey()";
```

Comment:
Can't we define this constant somewhere else? This constant seems like a random constant in this particular class. Even the one above it.

Reply:
Changed.